.github/workflows/build_and_test.yml

```
@@ -111,6 +117,7 @@ jobs:
           GH_REF: ${{ github.ref }}
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
         run: |
+          ls rskj-core/build/
           if [ "$GH_EVENT" = pull_request ]; then
```

```suggestion
          if [ "$GH_EVENT" = pull_request ]; then
```

I think we can remove this line, as this is for debugging purposes only

---

addressed